### PR TITLE
The first milestone: Changed the code as per required for documentation and some fixes

### PR DESCRIPTION
**Pull request by Tingyu-YE96**
Changed the code as per required for documentation and some fixes

### CODE REVIEW

File: src/game/game.c [hunk 1]
```diff
@@ -1,23 +1,187 @@
...
+	
+	if (is_full == board_size*board_size){
+		printf("!!! The board is full !!!\n\tTIE\n");
+		exit(0);
+	}
 }
 
-/*
- * Todo: implement player_move and complete the description for this function, after completed, please delete todo.
- * @Author: Tingyu Ye,
```
Comment (Mangosata) [inline src/game/game.c:16]: Please keep the comment, and fill the description

File: src/game/game.c [hunk 2]
```diff
@@ -1,23 +1,187 @@
 #include "../../include/game/game.h"
 
-/*
- * Todo: implement player_move and complete the description for this function, after completed, please delete todo.
- * @Author: Tingyu Ye,
```
Comment (Mangosata) [inline src/game/game.c:5]: Please keep the comment, and fill the description

**Review by Mangosata — COMMENTED** [review]
good job. but need to write some comments here. As the requirements, every function needs to have a brief description and a detailed description.

File: src/game/game.c (hunk 2 above)
Comment (Tingyu-YE96) [reply src/game/game.c:5]: I have commented the game.c

**Review by Tingyu-YE96 — COMMENTED** [review]
I have commented the code

**Comment by Mangosata** [comment]
good job. I will merge it into a new console branch.

**Comment by zhhzh19951215** [comment]
agree to merge

**Comment by Mangosata** [comment]
I have already created a new branch (console_version) for this console version. The development branch will be for the graphic version.

**Pull request closed without merging**